runtime/queries/sshclientconfig/highlights.scm

```diff
@@ -0,0 +1,324 @@
+(host) @keyword
+(host_value) @identifier
+
+(match) @keyword
+(match_value) @identifier
+
+(add_keys_to_agent) @keyword
+(add_keys_to_agent_value) @boolean
```

Comment:
I think this should be a separate pull request.

Also, it seemed kinda hard to manage this if we have to color each keyword manually, shouldn't it have be grouped?

Reply:
Opened #2498